apiary.apib

```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
 
 # Group aiakos v1
 
+## Regions [/v1/support]
+
+Get the region list with keys
```

Comment:
This is the description of the GET operation, not the description of the Regions resource, you should move it bellow line 47